clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -87,67 +124,113 @@ class AzureResourceManagerClient extends AzureBaseClient {
     resourceGroup
   }
 
-  boolean resourceGroupExists(AzureCredentials creds, String resourceGroupName) {
-    this.getResourceManagementClient(creds).getResourceGroupsOperations().checkExistence(resourceGroupName).isExists()
+  /**
+   *
+   * @param creds
+   * @param resourceGroupName
+   * @return
+     */
+  boolean resourceGroupExists(String resourceGroupName) {
+    client.getResourceGroupsOperations().checkExistence(resourceGroupName).body
```

Comment:
just need confirmation here; the method expects to return a boolean but we instead return a ResourceGroup. Is groovy properly handling the conversion? Can I presume a "null" ResourceGroup translate into "false" and anything else into "true"